src/components/jobDetailsComponent/userButtons.tsx

```diff
@@ -39,8 +39,7 @@ const UserButtons: React.FC = () => {
     }
     return new Promise((res, rej) => {
       try {
-        // res(dispatch(findOrCreateChat(job.id, job.userId, user.id)));
-        res(findOrCreateChat(job.id, job.userId, user.id));
+        res(dispatch(findOrCreateChat(job.id, job.userId, user.id)));
```

Comment:
this doesn't need to be dispatched if we're just using this to history.push(/inbox/id), the logic to get the messages should be on the chatroom component

Reply:
needs to be dispatched, i changed the find or create chat functionality a little bit